tests/x11/reboot_and_install.pm

```diff
@@ -15,14 +15,16 @@ use strict;
 
 use testapi;
 use utils 'reboot_gnome';
-use bootloader_setup qw(select_bootmenu_option bootmenu_default_params);
+use bootloader_setup;
+use registration;
 
 sub run() {
     reboot_gnome;    # reboot from previously booted hdd to do pre check or change e.g. before upgrade
-    select_bootmenu_option;
+    return if select_bootmenu_option == 3;
```

Comment:
it is evaluation of  lib/bootloader_setup   
###
if (get_var('ZDUP') || get_var("ONLINE_MIGRATION")) {                                                                                     send_key 'ret';    # boot from hard disk                                                                                              
return 3;
###